projects/ffmpeg/build.sh

```diff
@@ -219,6 +219,7 @@ fi
         --disable-doc \
         --disable-programs \
         --enable-demuxers \
+        --samples=fate-suite/ \
```

Comment:
This is needed otherwise we get a `warning: only a subset of the fate tests will be run because SAMPLES is not specified`

Reply:
how much time does this add to the test btw?

Reply:
Just re-run without the flag and got basically the same time as before (modulo some variance). I would say negligible.
```
real    0m53.840s
user    2m26.614s
sys     1m17.468s
```